libs/remix-simulator/src/VmProxy.ts

```diff
@@ -4,9 +4,9 @@ import { helpers } from '@remix-project/remix-lib'
 const  { normalizeHexAddress } = helpers.ui
 import { ConsoleLogs, hash } from '@remix-project/remix-lib'
 import BN from 'bn.js'
-import { isBigNumber } from 'web3-utils'
 import { toChecksumAddress, bufferToHex, Address, toBuffer } from '@ethereumjs/util'
 import utils from 'web3-utils'
+import { toBN } from 'web3-utils-legacy'
```

Comment:
This was a temporary decision made to reduce the friction of migrating to `4.x` since `BN` is no longer used

Reply:
I've removed this dependency